### ISSUE TITLE
Apple M1 support

### ISSUE THREAD

**Issue by heidihoward** [mwhittaker/quorums#1]
Quoracle does not work out of the box on the new Apple M1s because it depends on [numpy](https://github.com/numpy/numpy) which does not have M1 support. After trying various workaround (including [running terminal through rosetta](https://medium.com/better-programming/5-things-i-have-learned-when-using-the-m1-chip-macbook-air-a77f93c50381)), [this solution](https://stackoverflow.com/a/65581354) worked for me.

No action needed, I just wanted to document this workaround until numpy is updated.

**Comment by mwhittaker** [comment]
Oh wow, thanks for figuring this out and finding a solution Heidi! Hopefully numpy gets updated soon.

Outside of some code that makes plots, quoracle only really uses numpy in [one spot](https://github.com/mwhittaker/quoracle/blob/6e6f8103a89dfaeaac7118855bcff898ce983bf6/quoracle/quorum_system.py#L625-L631), so we can probably rewrite the code a little to use pure Python and remove numpy as a dependency entirely. I'll add that to my TODO list :)

**Comment by mwhittaker** [comment]
In commits 6bb6eb29143827d0db2701426d3a14a2cde69278, 9f0532a7d53f96dc25bdcde9c56d2fea9bfa6f4f, and a4dfc6f7515f332b5eafd32f737778120c822ea0, I removed the dependency on numpy. We still depend on matplotlib, though, and I think matplotlib depends on numpy, so trying to `pip install quoracle` on an Apple M1 will probably still fail I'm guessing. For that though, I think we just have to wait for things to get fixed upstream :)

**Comment by heidihoward** [comment]
Agreed, it should just be a matter of time before numpy is updated for M1 :)
